new_src/web/streamlit_app.py

```
@@ -71,6 +93,42 @@ def get_agent_response(user_input):
+                os.remove(old_path)
+
+        # 새 파일 저장
+        with open(file_path_on_disk, "wb") as f:
+            f.write(uploaded_file.getbuffer())
+        
+        st.session_state['uploaded_file_name'] = uploaded_file.name
+        
+    except ValueError as ve:
+        st.session_state['uploaded_file_name'] = None
+        st.error(f"파일 업로드 실패 (내용 오류): {ve}")
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)